src/modules/hoc/with-block-closer.js

```diff
@@ -15,7 +15,7 @@ export const intercept = e => e.stopPropagation();
 
 export default ( WrappedComponent ) => {
 	/**
-	 * Prevents clicks on block or blacklisted DOM elements
+	 * Prevents clicks on block or blocked DOM elements
```

Comment:
This should maybe be "disallowed" since we're talking about Gutenberg blocks in the same sentence?

Reply:
```suggestion
	 * Prevents clicks on block or disallowed DOM elements
```